### PR TITLE
now remote host has moved down to common layer and .Next dll gets loa…

### PR DESCRIPTION
**Pull request by heejaechang**
…ded if remote host option is on, we don't need this unnecessary indirection to delay load .Next. it will still not load .Next if remote host option is off.

### CODE REVIEW

**Comment by heejaechang** [comment]
@panopticoncentral @CyrusNajmabadi @KevinH-MS @mavasani @srivatsn can you take a look?

File: src/VisualStudio/Core/Def/Implementation/Diagnostics/VisualStudioDiagnosticAnalyzerExecutor.cs [hunk 1]
```diff
@@ -9,13 +9,44 @@
...
+using Microsoft.CodeAnalysis.Options;
+using Roslyn.Utilities;
+using Microsoft.CodeAnalysis.Execution;
+using System.Collections.Immutable;
+using System;
+using System.Linq;
+using System.IO;
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis.Remote.Diagnostics;
+using Microsoft.CodeAnalysis.Diagnostics.Telemetry;
```
Comment (CyrusNajmabadi) [inline src/VisualStudio/Core/Def/Implementation/Diagnostics/VisualStudioDiagnosticAnalyzerExecutor.cs]: sort

**Comment by panopticoncentral** [comment]
:shipit:

**Comment by heejaechang** [comment]
retest windows_release_unit32_prtest  please

**Comment by heejaechang** [comment]
retest due to d15 dll overwriting d14 dlls issue.

**Pull request merged**